x-pack/plugins/security_solution/server/lib/detection_engine/signals/rule_status_service.test.ts

```diff
@@ -87,43 +88,49 @@ describe('buildRuleStatusAttributes', () => {
 
 describe('ruleStatusService', () => {
   let currentStatus: ReturnType<typeof exampleRuleStatus>;
-  let ruleStatusClient: ReturnType<typeof ruleStatusSavedObjectsClientMock.create>;
+  let ruleStatusClient: ReturnType<typeof RuleExecutionLogClient>;
   let service: RuleStatusService;
 
   beforeEach(async () => {
     currentStatus = exampleRuleStatus();
-    ruleStatusClient = ruleStatusSavedObjectsClientMock.create();
+    ruleStatusClient = new RuleExecutionLogClient();
```

Comment:
(nit) Might want to put `Mock` in the name of this client just to make it more clear that it's a mock.

Reply:
++ it would make sense to rename it to something like `ruleExecutionLogClientMock.create` for consistency's sake.